src/main/java/com/suse/saltstack/netapi/calls/modules/Match.java

```diff
@@ -0,0 +1,61 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.LocalCall;
+import com.suse.saltstack.netapi.utils.ClientUtils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * salt.modules.match
+ */
+public class Match {
+
+    public static LocalCall<Boolean> compound(String tgt, Optional<String> minionId) {
```

Comment:
It does not make sense to document the module functions since they are already documented on the salt side. The variable names are borrowed from the documentation page as well to keep it consistent.  The only thing that may vary is the types if we can provide something saner on the java side.